src/neuroglancer/layer_panel.ts

```diff
@@ -258,8 +258,9 @@ export class LayerPanel extends RefCounted {
   constructor(
       public display: DisplayContext, public manager: LayerListSpecification,
       public viewerNavigationState: LinkedViewerNavigationState,
-      public selectedLayer: SelectedLayerState, public getLayoutSpecForDrag: () => any) {
-    super();
+      public selectedLayer: SelectedLayerState, public getLayoutSpecForDrag: () => any,
+      public displayLayerItemValue: WatchableValueInterface<boolean>) {
+    super(); 
     this.registerDisposer(selectedLayer);
```

Comment:
Use e.g.:

this.registerDisposer(displayLayerItemValue.changed.add(() => this.handleLayerValuesChanged()));

so that the display updates as soon as the config setting changes.

Reply:
Done.